lib/registries/addon/components/registries-search-result/template.hbs

```diff
@@ -21,22 +20,26 @@
             {{/if}}
         </h4>
 
-        {{#comma-list result.contributors filter=contribFilter as |contrib|}}
-            {{contrib.users.citedAs}}
-        {{/comma-list}}
+        <ul local-class="Contributors">
+            {{#each this.contributors as |contrib|}}
+                <li>
+                    {{#if contrib.link}}
+                        <a href={{contrib.link}}>{{contrib.name}}</a>
```

Comment:
We'll eventually want this to be a `link-to` (or equivalent). I guess we'll parse out the route from the OSF URL in identifiers or something?

Reply:
It'll probably look similar to the title? It'd be nice if we had a link-to that checked feature flags at somepoint